howso/react_aggregate.amlg

```diff
@@ -677,41 +767,40 @@
 		(if (= conditioned_residuals (true))
 			(let
 				(assoc
-					action_condition_precision (if
-						(= (get details "action_condition_precision") (null) )
-						"exact"
-						(get details "action_condition_precision")
-					)
-					context_condition_precision (if
-						(= (get details "action_condition_precision") (null) )
-						"exact"
-						(get details "action_condition_precision")
-					)
+					action_condition_precision
+						(if (= (get details "action_condition_precision") (null) )
+							"exact"
+							(get details "action_condition_precision")
+						)
+					context_condition_precision
+						(if (= (get details "action_condition_precision") (null) )
+							"exact"
+							(get details "action_condition_precision")
+						)
 				)
 				(assign
```

Comment:
(assign (assoc)) one line please. I see this isn't from this PR, but I would still appreciate it